apps/website/src/components/EditPageLink.astro

```diff
@@ -18,14 +20,31 @@ if (page === 'components') {
 const repoUrl = 'https://github.com/iTwin/iTwinUI';
 const githubEditUrl = `${repoUrl}/edit/main/apps/website/src/pages/docs/${fileName}`;
 ---
+<div>
 
-<a href={githubEditUrl} {...rest}>
-  <EditIcon aria-hidden='true' />Edit this page on GitHub
-</a>
+  <a href={githubEditUrl} {...rest}>
+    <EditIcon aria-hidden='true' />Edit
+  </a>
+
+  <a href='https://github.com/iTwin/iTwinUI/discussions/new?category=documentation' {...rest}>
+    <ChatIcon aria-hidden='true' />Provide feedback
+  </a>
+
+  <a href='https://github.com/iTwin/iTwinUI/discussions/categories/documentation' {...rest}>
+    <ChatsIcon aria-hidden='true' />View pending feedback
+  </a>
+
+</div>
 
 <style lang='scss'>
+
+  div {
+    display: flex;
+    gap: .25em 1em;
+    flex-wrap: wrap;
+    margin-top: .5em;
```

Comment:
you can use our tokens, like `var(--space-1)` etc

Reply:
No longer using a wrapper, but if we re-introduce a wrapper and need spacing I will use our spacing tokens.